pkg/utils/apiutil/apiutil.go

```diff
@@ -336,6 +336,22 @@ func ParseKey(name string, input map[string]any) ([]byte, string, error) {
 	return returned, rawKey, nil
 }
 
+// ParseHexKeys decodes hexadecimal keys to bytes if the format is "hex".
+func ParseHexKeys(format string, keys []string) (hexStrings []string, err error) {
+	if format != "hex" {
+		return keys, nil
+	}
```

Comment:
Shall we move it out of this function?

Reply:
Since the function is named ParseHexKeys, I think it has already be in hex format before calling this function.